frontend/src/app/acrobatics/acrobatics.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+
+export interface Acrobatic {
+  id: number;
+  name: string;
+  icon: string;
+  difficulty: number;
+}
+
+@Component({
+  selector: 'app-acrobatics',
+  templateUrl: './acrobatics.component.html',
+  styleUrls: ['./acrobatics.component.scss'],
+})
+export class AcrobaticsComponent {
+  listOfAcrobatics: Acrobatic[] = [
+    { id: 1, name: 'Jump', icon: 'settings_accessibility', difficulty: 2 },
```

Comment:
**question:** no need to do anything but any thoughts on how this kind of names would be localized?

Reply:
I assume you're referring to the property `name: 'Jump'`. These could be localized in the template in the same way as other texts (e.g `{{'ACROBATICS.' + name | uppercase | translate}}`. However, this could become burdensome if the property is accessed a lot in the template. Perhaps the translation could be done when the name is initialized (using the translation service) so that it is simpler to use.